src/math.ts

```diff
@@ -40,7 +38,7 @@ export const calculateSecondsInside = (
     secondsPerLiquidityInsideInitial
   )
     .mul(liquidity)
-    .mul(SECONDS_INSIDE_DENOMINATOR)
+    .mul(POINTS_DENOMINATOR)
```

Comment:
That's not correct. You are `calculatingSecondsInside` so it shouldn't have Points scale, but a seconds type denominator that we utilized earlier. The Points scale should be applied when calculating points
```ts
  const points = pointsToDistribute
    .mul(secondsInside)
    .mul(POINTS_DENOMINATOR)
    .div(totalSecondsPassed)
    .div(SECONDS_INSIDE_DENOMINATOR);
    ```